dandi/dandiapi.py

```diff
@@ -1106,6 +1106,10 @@ def wait_until_valid(self, max_time: float = 120) -> None:
             v = self.get_version(self.version_id)
             if v.status is VersionStatus.VALID and not v.asset_validation_errors:
                 return
+            # TODO(asmacdo) can we fail fast?
```

Comment:
This should be removed from this PR-- but without this each invalid dandiset waits for 2 minutes before failing.